src/store_client.cc

```diff
@@ -499,6 +513,9 @@ store_client::readBody(const char *, ssize_t len)
 {
     int parsed_header = 0;
 
+    if (ale && len > 0 && flags.disk_io_pending)
```

Comment:
What about the initial readBody() call from readHeader() where flags.disk_io_pending is false? Or we expect readBody() to be called again with the same buffer?

Reply:
> I missed that we count it in readHeader() itself...

Yes, we do. The flags.disk_io_pending is there to prevent double-counting. There is already a comment (right below this code) about disk_io_pending being false when called by readHeader() so I decided not to add another comment.

This change request is addressed, right?